openlibrary/macros/databarWork.html

```diff
@@ -51,8 +51,7 @@ <h3 class="header">
         $ edition = page.works and page
         $ lists_widget = render_template('lists/widget', edition or work, include_rating=True, include_header=False, include_widget=True, include_showcase=False, user_lists_only=False)
 
-        $# For the Works & Editions page, always use ground_truth_availability API for selected Edition
-        $:macros.LoanStatus(page, allow_expensive_availability_check=True, secondary_action=editions_page, check_sponsorship=editions_page, sponsorship_help=editions_page, check_loan_status=editions_page, post=lists_widget)
+        $:macros.LoanStatus(page, secondary_action=editions_page, check_sponsorship=editions_page, sponsorship_help=editions_page, check_loan_status=editions_page, post=lists_widget)
```

Comment:
I think we may want to adapt this so if the `page.availability["status"]` if `error` or `private` we still do the more expensive check. This shouldn't affect most books and will result in the correct result being shown in the cases which are most likely to be wrong from Bulk Availability API.

Reply:
I think this should be resolved w/ the latest change I made